exporter/datadogexporter/config.go

```diff
@@ -279,6 +279,10 @@ type TracesConfig struct {
 	// If the overhead remains high, it will be due to a high cardinality of `peer.service` values from the traces. You may need to check your instrumentation.
 	PeerServiceAggregation bool `mapstructure:"peer_service_aggregation"`
 
+	// TraceBuffer specifies the number of Datadog Agent TracerPayloads to buffer before dropping.
+	// The default value is 0, meaning the Datadog Agent TracerPayloads are unbuffered.
+	TraceBuffer int `mapstructure:"trace_buffer"`
```

Comment:
Does this option come from the Datadog Agent? If not, what about `trace_buffer_size` or even `buffer_size`?  (since we now we are talking about traces) Or even a new `buffer` section with a `size` option?

Reply:
Yes, this is from Datadog Agent: https://github.com/DataDog/datadog-agent/blob/612878f2e1774e8e784b140aa96f54c764adee14/comp/trace/config/setup.go#L275, so may be better to be consistent here.